analyzers/its/config/Ember-MM/SonarLint.xml

```diff
@@ -31,6 +31,7 @@
     <Rule><Key>S4829</Key></Rule>
     <Rule><Key>S4834</Key></Rule>
     <Rule><Key>S5042</Key></Rule>
+	<Rule><Key>S5332</Key></Rule>
```

Comment:
pedantic: the sonarlint.xml files use spaces instead of tabs, so it would be nice to change the tabs to spaces in them.